ister_gui.py

```diff
@@ -1935,6 +1949,14 @@ def handler(self, config):
             return exc
         return self._action
 
+    def _erase_dup_mount(self, point, part):
+        for idx, chc in enumerate(self.choices):
+            if '{} '.format(point) in chc and '{} '.format(part) not in chc:
```

Comment:
this is done to avoid finding the `/` mount point inside the `/boot` mountpoint and to make sure we are matching the entire partition name instead of a portion of another.